code/modules/mob/living/carbon/human/species_types.dm

```diff
@@ -549,6 +554,7 @@ var/global/list/synth_flesh_disguises = list()
 	SA_para_min = 0
 	SA_sleep_min = 0
 	dangerous_existence = 1
```

Comment:
While you're here, could you remove dangerous_existence from synths. They were only added here to keep them blacklisted from stuff that you've now made a better var to use for that. It's not actually dangerous at all to be a synth